### ISSUE TITLE
Add FilterToolbar to Select VMs step of wizard

### ISSUE THREAD

**Issue by mturley** [konveyor/virt-ui#131]
Should be done after #129 is merged, but I didn't want to further clutter that huge PR.

Similarly to #125, we need to add filters to the Select VMs step of the wizard as well.

@vconzola which columns do you want us to filter on there, all of them? is it ok if we don't filter by the Migration Assessment, since that data can be complex? We could have a Select dropdown filter that matches based on worst severity present in each VM, but that feels low priority.

![Screen Shot 2020-10-14 at 2 04 57 PM](https://user-images.githubusercontent.com/811963/96027704-42f79500-0e26-11eb-9618-08de71419a24.png)

**Comment by vconzola** [comment]
@mturley Assessment is probably the column people will want to filter on the most, since it tells them which VMs are "eligible" for migration. Tell me more about the complexity of the data. The assessment shown is just the worst severity for each VM.

**Comment by mturley** [comment]
Ok, we can do that. It's only complex in the sense that the API objects are not flat "worst severity" values, so we just need to convert the list of conditions for each VM into one worst condition before running it through the filter. No problem, I'll quit trying to cut corners :)
